airbyte-integrations/connectors/source-jira/source_jira/manifest.yaml

```diff
@@ -38,6 +38,12 @@ definitions:
       type: DpathExtractor
       field_path: ["{{ parameters.extract_field }}"]
 
+  selector_base:
```

Comment:
Why did we need to add this and can't rely on the interpolation of `{{ parameters.extract_field }}` anymore?

Reply:
It's due to changes in `JsonDecoder`. Before decoder used to return all json response like: `[{entity 1}, {entity 2}]`. So to extract records path ["*"] were used. But now records are being yielded one by one from response by `JsonDecoder`. So path ["*"] works incorrectly in this case and we just need to keep path empty to return record from JsonDecoder

Reply:
I think you are referring to [this PR](https://github.com/airbytehq/airbyte/pull/38829) right? When checking this one, it seems like [the * parameter](https://github.com/airbytehq/airbyte/pull/38829/files#diff-54c832113b872fcf5b9a42fffcf84e45fe2c9e5384cd9af012b84648a23ba747R73-R74) is still supported. From what you mention though, the behavior has changed, is that correct? Can you share a snippet of code or a test that I can perform on my side to understand this change?

Reply:
I'm referring to [this](https://github.com/airbytehq/airbyte/pull/38829/files#diff-404f304b08b4a327259c7999f75283e1f111edaac2bb09196cb2c536f141f2deR32) change in json decoder class. 
We don't need` "*" ` anymore, as dpath receives record in the loop [here](https://github.com/airbytehq/airbyte/pull/38829/files#diff-54c832113b872fcf5b9a42fffcf84e45fe2c9e5384cd9af012b84648a23ba747R68). 
You can reproduce it by adding` "*" `in base selector and related streams will fail.

Reply:
Ok, so basically, both works but since we don't need the dpath evaluation, we prefer to remove that `*` is that right? 

Knowing this, I'm just wondering about the added complexity of the manifest of having this. If the performance cost is low, I think I would keep the `*` as it seems more maintainable. However, I don't have a strong opinion so I'll let you decide